monkestation/code/datums/traits/negative.dm

```diff
@@ -99,3 +99,32 @@
 		message = replacetext(message, "r", "w")
 		message = replacetext(message, "l", "w")
 	speech_args[SPEECH_MESSAGE] = message
+
+/datum/quirk/nugget
+	name = "Nugget"
+	desc = "An accident caused you to lose ALL of your limbs. There's no way your insurance payed for all those prosthetics!"
+	value = -1
+	var/slot_string = "limb"
+
+/datum/quirk/nugget/on_spawn()
+	var/mob/living/carbon/human/nuggeted = quirk_holder
+	for(var/obj/item/bodypart/BP in nuggeted.bodyparts)
```

Comment:
i'd personally prefer if this was called something like bodypart and not BP, same reasoning as single letter variables

Reply:
Good catch